content/blog/global-case-study.md

```diff
@@ -16,76 +16,82 @@ A leading comprehensive global mental health solution for employers. It started
 
 ## Global Care
 
-The next-generation tech-enabled precision mental healthcare company went international, localizing the experience and bringing quality care to 41 countries in 22 new languages. While it continues to extend the care globally, let's take a look at the technical how.
+The next-generation tech-enabled precision mental healthcare company went international, localizing the experience and bringing quality care to 41 countries in 22 new languages.
+
+To extend the care globally, they had to solve some challenges, which included internationalization of the application along with content localization.
 
 ## What We Did
 
-- Created a framework to enable global offerings in the web application
-- Country specific language content
-- Streamline workflow integrating a third-party translation service~
-- Extended international offering to 40 more countries in 9 more languages
-- Upgraded major project dependencies to access internationalization packages
+- Created a framework to globalize care services
+- Provide country specific localized content
+- Streamline and automate translation workflow, integrating a third-party translation service
```

Comment:
Should we do past tense here too? Run Grammarly please
By integrating a 3rd party translation service we streamlined and automated translation workflow to minimize manual work every time we introduced a new region and a language